tests/unit/utils.py

```diff
@@ -0,0 +1,25 @@
+import os
+import tarfile
+from pathlib import Path
+
+
+def _check_tar_members(tarpath: Path, tarmembers: list[tarfile.TarInfo], dest_path: Path) -> None:
```

Comment:
I think a short docstring explaining what is being checked would be helpful here.